programs/wen_wns_marketplace/src/state/listing.rs

```diff
@@ -0,0 +1,23 @@
+use anchor_lang::prelude::*;
+
+#[account]
+pub struct Listing {
+    pub bump: u8,
+    pub mint: Pubkey,
+    pub payment_mint: Pubkey,
+    pub seller: Pubkey,
+    pub seller_token_account: Pubkey,
+    pub listing_amount: u64,
+}
+
+impl Listing {
+    pub fn size() -> usize {
```

Comment:
Should be able to use the init_space macro instead of manually coding this

Reply:
Macros just generate more code than intended. Since this was a trivial program, it was right for me to go ahead with a simple size implementation function.